### DISCUSSION TITLE
Could not resolve host: api-v2.fattureincloud.it

### DISCUSSION THREAD

**Discussion by paolocerto** [fattureincloud/api#272] [Q&A] [answered]
Hello,

this is the error I get when I try to get the access token
Uncaught GuzzleHttp\\Exception\\ConnectException: cURL error 6: Could not resolve host: api-v2.fattureincloud.it

What can I do?
I recently updated GuzzleHttp and Fattureincloud API in composer.
In a first test it worked. After two weeks it doesn't.
Thank you for your help
Regards,
Paolo

**Comment by valmoz** [comment]
Hi @paolocerto 
I just tested, and the https://api-v2.fattureincloud.it endpoint can be reached correctly.
You can also test it from the [API Reference](https://developers.fattureincloud.it/api-reference).

Maybe you forgot to add "https"? Also, can you please check if you're able to reach the endpoint from your machine? (for example from command line?)

Thanks
Mauro

**Comment by paolocerto** [comment, marked as answer]
Hi @valmoz 
thank you for the response
I use the PHP SDK, so I no need to set the URL.

I just re-run the test and the test works as expected.
Probably there is a difference in the production code. 
I'm searching...

Thank you again
Regards,
Paolo
